hack/common.sh

```diff
@@ -132,7 +131,7 @@ get_version() {
   GIT_COMMIT=$(git rev-parse --short HEAD)
   VERSION="${OPERATOR_VERSION}+${GIT_COMMIT}"
 
-  if [ -n "$(git status --porcelain)" ]; then
+  if [ -n "$(git status version tools.go go.mod go.sum vendor build cmd hack pkg --porcelain)" ]; then
```

Comment:
Missing `Makefile` here.

Reply:
Ah!! Thanks for catching !

Reply:
I can change the way the operator is built by modifying the `Makefile`. It needs to be present here. Moreover you don't get anything by **not** adding it here.